.circleci/config.yml

```diff
@@ -181,26 +202,35 @@ workflows:
           context:
             - hypertrace-vulnerability-scanning
             - dockerhub-read
-      - publish:
+      - merge-publish:
           context:
             - hypertrace-publishing
             - dockerhub-read
           requires:
             - build
-            - integration-test
```

Comment:
This should be a publish prerequisite, but appears to be removed

Reply:
addressed.